flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/api/TableConfigOptions.java

```diff
@@ -31,16 +31,9 @@
 	// ------------------------------------------------------------------------
 	//  Source Options
 	// ------------------------------------------------------------------------
-
-	public static final ConfigOption<Boolean> SQL_EXEC_SOURCE_VALUES_INPUT_ENABLED =
-			key("sql.exec.source.values-input.enabled")
-					.defaultValue(false)
-					.withDescription("Whether support values source input. The reason for disabling this " +
-									"feature is that checkpoint will not work properly when source finished.");
-
-	public static final ConfigOption<Long> SQL_EXEC_SOURCE_IDLE_TIMEOUT =
-			key("sql.exec.source.idle.timeout.ms")
-					.defaultValue(-1L)
+	public static final ConfigOption<String> SQL_EXEC_SOURCE_IDLE_TIMEOUT =
```

Comment:
Is this already checked?

Reply:
Thanks, have checked, its description is correct.